Server/datalayer/user.datalayer.js

```diff
@@ -8,6 +8,49 @@ class usersDatalayer {
     * Initializes the datalayer
     */
 
+    // Creates new user 
+    static createUser(reqbody) {
```

Comment:
Don't use `req` word here

Reply:
Removed and updated with userObj as suggested above